### ISSUE TITLE
Release .NET linter to marketplace

### ISSUE THREAD

**Issue by ahanoff** [wearerequired/lint-action#417]
hi @ocean90, I've decided to run new .NET format linter from #386 

```yml
Run wearerequired/lint-action@v1
  with:
    dotnet_format: true
    github_token: ***
    continue_on_error: true
    auto_fix: false
```

but I'm getting error: `Warning: Unexpected input(s) 'dotnet_format', valid inputs are...`

Latest release on Marketplace (https://github.com/marketplace/actions/lint-action) is v1.11.1. Despite its readme includes new C# linter, action code is not there yet.

Could you please release it if everything is all right? Thanks in advance

**Comment by ocean90** [comment]
Thanks for the reminder, released in [v1.12.0](https://github.com/wearerequired/lint-action/releases/tag/v1.12.0).
